spec/spec_helper.rb

```diff
@@ -22,7 +22,7 @@
 VCR.configure do |config|
   # Uncomment the line below to record new VCR cassettes.
   # When this is commented out, VCR will reject all outbound HTTP calls.
-  # config.allow_http_connections_when_no_cassette = true
+  config.allow_http_connections_when_no_cassette = true
```

Comment:
Make sure to comment this back out so HTTP calls are not made.

Reply:
That's been commented-out again.